pull_request_template.md

```diff
@@ -8,13 +8,19 @@
 [Discuss the impact of your changes on the project. This might include effects on performance, new dependencies, or changes in behaviour.]
 
 ## Checklist
-- [ ] My code adheres to the coding and style guidelines of the project.
-- [ ] I have performed a self-review of my own code.
-- [ ] I have extended current unit tests suite.
-- [ ] I have made corresponding changes to the documentation.
-- [ ] My changes generate no new warnings or new Sonar issues
-- [ ] I have passed the PR linting stage
-- [ ] I contacted my SRE colleague to update the configuration (if any changes)
+- [ ] **[JIRA]** I have linked the JIRA ticket to the PR.
+- [ ] **[CODE]** My code adheres to the coding and style guidelines of the project.
+- [ ] **[REVIEW]** I have performed a self-review of my own code.
+- [ ] **[DOCS]** I have made corresponding changes to the documentation.
+- [ ] **[SONAR]** My changes generate no new warnings or new Sonar issues.
+- [ ] **[LINT]** I have passed the PR linting stage.
+- [ ] **[SRE]** I contacted my SRE colleague to update the configuration (if any changes)
+- [ ] **[LABELS]** I added the relevant labels to the PR.
+- [ ] **[DEPS]** I have added/updated external dependencies/packages.
+- [ ] **[DEPS]** I have updated runtime version (i.e.: from node19 to node20).

```

Comment:
Same as above, This one will be so rarely checked on other applications, I believe will have more value to extend the Additional information description and put in there if it's needed.

Reply:
The approach is a general one, if it applies for your repository you check it, it's not a concession on having or not having to fill it. After all we are not allowed to have our own individual PR template.

Reply:
in my opinion, having less is better, this template can be overwritten by each repository individually if there are multiple points to add

Reply:
I tend to agree with you though we must remember that it's laso the DEV team responsibility to update versions or version related information.